Source/WebCore/css/MediaQueryList.idl

```diff
@@ -16,13 +16,16 @@
  *  the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
  *  Boston, MA 02110-1301, USA.
  */
+
+// https://drafts.csswg.org/cssom-view/#the-mediaquerylist-interface
+
 [
     ActiveDOMObject,
     Exposed=Window
 ] interface MediaQueryList : EventTarget {
     readonly attribute DOMString media;
     readonly attribute boolean matches;
-
+    // FIXME: Rename 'listener' to 'callback' as per web specification.
```

Comment:
You can just safely rename it, the argument name isn't important.

Reply:
Renamed. Fixed in 'd99f558' commit.